jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.py

```diff
@@ -2,13 +2,28 @@
 
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import PluginTemplateMixin, DatasetSelectMixin
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.utils import PRIHDR_KEY, COMMENTCARD_KEY
 
 __all__ = ['MetadataViewer']
 
 
 @tray_registry('g-metadata-viewer', label="Metadata Viewer")
 class MetadataViewer(PluginTemplateMixin, DatasetSelectMixin):
+    """
+    See the :ref:`Metadata Viewer Plugin Documentation <imviz_metadata-viewer>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``dataset`` (:class:`~jdaviz.core.template_mixin.DatasetSelect`):
+      Dataset to expose the metadata.
+    * :attr:`show_primary`:
+      Whether to show MEF primary header metadata instead.
+    * :attr:`metadata`

```

Comment:
```suggestion
    * :attr:`metadata`:
      Read-only metadata. If the data is loaded from a multi-extension FITS file,
      this can be the extension header or the primary header, depending on
      ``show_primary`` setting.

```